lab2.js

```diff
@@ -118,8 +178,19 @@ assert((new Human()).sayHello(new Klingon()) === 'nuqneH',
 // will test your code)
 //*********************************************************
 
+var fruit = ['cherry', 'banana', 'apple'];
+var sortedFruit = ['banana', 'apple', 'cherry'];
+
 function lastLetterSort(stringArray) {
   function byLastLetter(a, b) {
+    if (a.slice(-1) < b.slice(-1)) {
+      return -1;
+    } else if (a.slice(-1) > b.slice(-1)) {
+      return 1;
+    } else {
+      return 0;
```

Comment:
you can also return the difference between the two values. As long as sort gets a positive number, a negative number, or 0, it's happy.